tests/topotests/bgp_6vpe_ebgp_topo1/pe1/ipv6_routes_vrf.json

```diff
@@ -0,0 +1,142 @@
+{
+  "fd00:100::/64": [
+    {
+      "prefix": "fd00:100::/64",
+      "protocol": "connected",
+      "vrfName": "vrf1",
+      "selected": true,
+      "destSelected": true,
+      "distance": 0,
+      "metric": 0,
+      "installed": true,
+      "nexthops": [
+        {
+          "fib": true,
+          "directlyConnected": true,
+          "interfaceName": "eth-h1",
+          "active": true
+        }
+      ]
+    }
+  ],
+  "fd00:200::/64": [
+    {
+      "prefix": "fd00:200::/64",
+      "protocol": "bgp",
+      "vrfName": "vrf1",
+      "selected": true,
+      "destSelected": true,
+      "distance": 20,
+      "metric": 0,
+      "installed": true,
+      "nexthops": [
+        {
+          "ip": "::ffff:c000:205",
+          "afi": "ipv6",
+          "vrf": "default",
+          "active": true,
+          "labels": [
+            102
+          ],
+          "weight": 1
+        }
+      ]
+    }
+  ],
+  "fd00:201::/64": [
+    {
+      "prefix": "fd00:201::/64",
+      "protocol": "bgp",
+      "vrfName": "vrf1",
+      "selected": true,
+      "destSelected": true,
+      "distance": 20,
+      "metric": 0,
+      "installed": true,
+      "nexthops": [
+        {
+          "ip": "::ffff:c000:205",
+          "afi": "ipv6",
+          "vrf": "default",
+          "active": true,
+          "labels": [
+            102
+          ],
+          "weight": 1
+        }
+      ]
+    }
+  ],
+  "fd00:300::/64": [
+    {
+      "prefix": "fd00:300::/64",
+      "protocol": "bgp",
+      "vrfName": "vrf1",
+      "selected": true,
+      "destSelected": true,
+      "distance": 20,
+      "metric": 0,
+      "installed": true,
+      "nexthops": [
+        {
+          "ip": "::ffff:c000:205",
+          "afi": "ipv6",
+          "vrf": "default",
+          "active": true,
+          "labels": [
+            102
+          ],
+          "weight": 1
+        }
+      ]
+    }
+  ],
+  "fd00:400::/64": [
+    {
+      "prefix": "fd00:400::/64",
+      "protocol": "bgp",
+      "vrfName": "vrf1",
+      "selected": true,
+      "destSelected": true,
+      "distance": 20,
+      "metric": 0,
+      "installed": true,
+      "nexthops": [
+        {
+          "ip": "::ffff:c000:205",
```

Comment:
per the RFC4291, the mapped address are dumped/displayed with the format ::ffff:A.B.C.D

Reply:
You are correct. The issue does not come from these patches. Should it be fixed in the same pull-request?

Reply:
IMO doing with a separate PR is fine, because it's a separate problem.